src/note.rs

```diff
@@ -86,6 +89,8 @@ pub struct Note {
     recipient: Address,
     /// The value of this note.
     value: NoteValue,
+    /// The type of this note.
+    note_type: NoteType,
```

Comment:
This could just be `asset: NoteAsset`.

Reply:
Thanks for the comments, we are working to make the fixes.

Are you suggesting change every occurrence of `note_type` to `asset` or only this field (which accounts for most of the occurrences) ?

Reply:
Ended up with `asset: AssetId` and `note_type` -> `asset` across the entire repo.